__tests__/__unit__/uss/USSTree.unit.test.ts

```diff
@@ -14,12 +14,18 @@ import { createUSSTree, USSTree } from "../../../src/uss/USSTree";
 import { ZoweUSSNode } from "../../../src/uss/ZoweUSSNode";
```

Comment:
It looks like all changes in this file are related to formatting. Perhaps it is time, that we implement prettier https://github.com/zowe/vscode-extension-for-zowe/issues/815